compiler/parser/RdrHsSyn.hs

```diff
@@ -2515,7 +2509,7 @@ mkQualType :: String -> LHsType GhcPs
 mkQualType = noLoc . HsTyVar NoExt NotPromoted . noLoc . qualifyDesugar . mkTcOcc
 
 mkQualClass :: String -> LHsType GhcPs
-mkQualClass = noLoc . HsTyVar NoExt NotPromoted . noLoc . qualifyDesugar . mkClsOcc
+mkQualClass = noLoc . HsTyVar NoExt NotPromoted . noLoc . qualifyDesugar . mkClsOccgit
```

Comment:
`mkClsOccgit` looks like you were typing in the wrong buffer

Reply:
Thanks.